pennylane/devices/tests/test_properties.py

```diff
@@ -265,3 +265,39 @@ def circuit():
         else:
             with pytest.raises(NotImplementedError):
                 circuit()
+
+    def test_supports_broadcasting(self, device_kwargs, mocker):
+        """Tests that the device reports correctly whether it supports parameter broadcasting
+        and that it can execute broadcasted tapes in any case."""
+        # pylint: disable=protected-access
+        device_kwargs["wires"] = 1
+        dev = qml.device(**device_kwargs)
+        cap = dev.capabilities()
+
+        assert "supports_broadcasting" in cap
+
+        @qml.qnode(dev)
+        def circuit(x):
+            if cap["model"] == "qubit":
+                qml.RX(x, wires=0)
+            else:
+                qml.Rotation(x, wires=0)
```

Comment:
The test suite currently only supports the qubit model.

Reply:
Ah, I did not look properly at the test output for that case, even tried it :sweat_smile: 
NB: Many tests differentiate between the two models, and include a CV version. Should I exclude it here?

Reply:
Oh :thinking: Strange, originally we used to only regard the qubit model and run the tests on such devices:
https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/devices/tests/__init__.py#L16

If the CV case is often considered, good to include.

Reply:
Okay, I'll leave it in for consistency, but maybe we should investigate whether it is unnecessary and can be removed everywhere, or whether the file docstring should be updated?

Reply:
Yes, that sounds good :+1: